index.html

```diff
@@ -190,8 +192,16 @@ <h2>
     def buildProtocol(self, addr):
         return Echo()
 
-endpoints.serverFromString(reactor, "tcp:1234").listen(EchoFactory())
-reactor.run()
+async def echo_server(reactor):
+    await endpoints.serverFromString(reactor, "tcp:1234").listen(EchoFactory())
+    await task.deferLater(reactor, math.inf, lambda: pass)
```

Comment:
I suggest dropping the `lambda` here as `deferLater` doesn't require it. It just adds an additional function call.

Reply:
```suggestion
    await task.deferLater(reactor, math.inf)
```